### DISCUSSION TITLE
An option to skip slowdown on failure

### DISCUSSION THREAD

**Discussion by szmarczak** [ppy/osu#14910] [Ideas] [answered]
I find the slowdown effect quite annoying. The thunder sound as well.

**Comment by bdach** [comment, marked as answer]
quick retry (press and hold <kbd>~</kbd>) is available even during the fail sequence. is that not sufficient?

**Reply by szmarczak** [reply]
It is, but unfortunately doesn't stop the thunder sound.

**Reply by szmarczak** [reply]
Another thing is that miss icons still pop up after you fail / it is still possible to click when the slowdown is happening.

**Reply by peppy** [reply]
You will eventually be able to skin the sound.

**Reply by bdach** [reply]
> miss icons still pop up after you fail / it is still possible to click when the slowdown is happening

that is [explicitly intended](https://github.com/ppy/osu/issues/12166#issuecomment-805457565)

**Reply by szmarczak** [reply]
@peppy Is there an issue to track this?

**Reply by szmarczak** [reply]
@bdach Regarding the locked issue:

> saying that you also experience it without providing any additional details doesn't really help us in any way.

Except I asked a question that remained unanswered:

> Perhaps this could be customizable via an option?

You could answer in many ways, e.g. `No, because ...` or `Yes, but we need more feedback` or anything more comprehensive.

I'll take this as a hard no then. If you still find this question spam, you can report me to GitHub.

Let's end this discussion with peace ✌🏼 Cheers.
